packages/cozy-stack-client/src/AppCollection.js

```diff
@@ -37,31 +35,6 @@ class AppCollection {
       next: false
     }
   }
-
-  find(...args) {
-    console.warn(
-      `find() method for doctype '${
-        this.doctype
-      }' relies internally on DocumentCollection.find() and may not fetch all expected data.`
-    )
-    return DocumentCollection.prototype.find.call(this, ...args)
-  }
-
-  async get() {
-    throw new Error('get() method is not yet implemented')
-  }
-
-  async create() {
-    throw new Error('create() method is not available for applications')
-  }
-
-  async update() {
-    throw new Error('update() method is not available for applications')
-  }
-
-  async destroy() {
-    throw new Error('destroy() method is not available for applications')
-  }
```

Comment:
`AppCollection` will use `find` from `DocumentCollection`, but also `get`, `create`, `update` and `destroy`. Maybe you want to keep these methods implementations that throw an error here?

Reply:
Mmmmh I am not sure what we really want here. But maybe we should keep throwing while we have not decided.